src/Ticket.java

```diff
@@ -43,56 +51,27 @@ public Ticket(String concertHall,
         this.time = time;
     }
 
-    public void setId(String id) {
-        if (id == null) {
-            throw new IllegalArgumentException("Identifier cannot be null");
-        } else if (id.length() > 4) {
-            throw new IllegalArgumentException("Identifier must not exceed 4 digits and/or characters.");
-        }
-        this.id = id;
-    }
-
-    public String getId() {
-        return id;
-    }
-
-    public void setConcertHall(String concertHall) {
-        if (concertHall == null) {
-            throw new IllegalArgumentException("Concert hall cannot be null");
-        } else if (concertHall.length() > 10) {
-            throw new IllegalArgumentException("Concert hall must not exceed 10 characters.");
-        }
-        this.concertHall = concertHall;
-    }
-
-    public void setEventCode(String eventCode) {
-        if (eventCode == null) {
-            throw new IllegalArgumentException("Event code cannot be null.");
-        } else if (eventCode.length() != 3 || !eventCode.matches("\\d{3}")) {
-            throw new IllegalArgumentException("Event code must be exactly 3 digits.");
-        }
-        this.eventCode = eventCode;
+    public void setTime(long time) {
+        this.time = time;
     }
 
-    public Sector getSector() {
-        return this.sector;
+    public void setSector(Sector sector) {
+        this.sector = sector;
     }
 
-    //**ability to save ticket price
     public void toSaveTicketPrice(double price) {
         BigDecimal ticketPrice = new BigDecimal(price);
         ticketPrice = ticketPrice.setScale(2, RoundingMode.HALF_UP);
         this.price = ticketPrice;
-          }
+    }
 
-    public String toStringTicketPrice() {
-       return "The ticket id: " + this.id + " price is: " + this.price;
+    public String printTicketPrice() {
+        return "The ticket id: " + this.id + " price is: " + this.price;
```

Comment:
you don't print. You just return String)) 
you can use void, and print inside this method)

Reply:
void done